cmd/aro/portal.go

```diff
@@ -158,7 +158,7 @@ func portal(ctx context.Context, log *logrus.Entry, audit *logrus.Entry) error {
 		return err
 	}
 
-	hostname := "localhost:8444"
+	hostname := "localhost:3000"
```

Comment:
nit: hostname vs. address could use a comment
Took me a while to make sure that hostname is the externally visible part of the URI (used for AAD redirects for now).

Reply:
added a comment that should explain why it's different